index.bs

```diff
@@ -1620,28 +1670,41 @@ To <dfn>serialize as a remote value</dfn> given a |value|, a |max depth|,
 1. In the following list of conditions and associated steps, run the first set
    of steps for which the associated condition is true:
 
+1. Let |handle id| be the [=handle for an object=] given |realm|, |ownership|
+   and |value|.
+
+1. Let |known object| be <code>true</code>, if |value| is in the
+   |serialization internal map|, otherwise <code>false</code>.
+
+1. Let |child ownership| be <code>none</code>.
```

Comment:
We usually write string literals like `"<code>none</code>"`.

Reply:
done